doc/examples/color_exposure/plot_rgb_to_hsv.py

```diff
@@ -0,0 +1,44 @@
+"""
+================
+RGB to HSV
+================
+
+This example converts an image with RGB channels into an image with HSV (Hue,
+Saturation,Value) channels.
+
+This coordinate system is useful for color-based thresholding since the color
+of a pixel is only described by its Hue value (and not the R,G,B values). [1]_
+
+References
+----------
+.. [1] https://en.wikipedia.org/wiki/HSL_and_HSV
+
+"""
+import matplotlib.pyplot as plt
+
+from skimage import data
+from skimage.color import rgb2hsv
+
+original = data.coffee()
```

Comment:
As your example is on conversion _from_ _RGB_, I suggest to rename `original` to `rgb_img`, and update the corresponding figure title.